### PR TITLE
[FLASK] `snaps-monorepo@0.32.2`

### PR DESCRIPTION
**Pull request by FrederikBolding**
## Explanation

Bumps Snaps packages to `0.32.2` and handles any breaking changes.

Also adds new URLs and metadata for execution environments and the snap registry.

### CODE REVIEW

**Review by GuillaumeRx — DISMISSED** [review]

File: app/scripts/metamask-controller.js [hunk 1]
```diff
@@ -688,6 +689,8 @@ export default class MetamaskController extends EventEmitter {
           `${this.approvalController.name}:hasRequest`,
           `${this.approvalController.name}:acceptRequest`,
           `${this.approvalController.name}:rejectRequest`,
+          `SnapController:getPermitted`,
+          `SnapController:install`,
```
Comment (hmalik88) [inline app/scripts/metamask-controller.js:693]: why not `${this.snapController.name}:getPermitted` and ${this.snapController.name}:install`?

**Review by david0xd — DISMISSED** [review]

File: app/scripts/metamask-controller.js (hunk 1 above)
Comment (FrederikBolding) [reply app/scripts/metamask-controller.js:693]: No reason, we can do that 👍

Comment (GuillaumeRx) [reply app/scripts/metamask-controller.js:693]: because `${this.snapController.name}` is undefined

Comment (FrederikBolding) [reply app/scripts/metamask-controller.js:693]: Oh yeah, we can't since the PermissionController is defined before the SnapController

Comment (FrederikBolding) [reply app/scripts/metamask-controller.js:693]: We could move it around, but I don't think that its worth the hassle right now

**Review by hmalik88 — DISMISSED** [review]
LGTM, let's not forget to move around the order and update with the controller name in the future :)

**Comment by FrederikBolding** [comment]
> LGTM, let's not forget to move around the order and update with the controller name in the future :)

We need a better way to do it I think, because SnapController requires PermissionController actions and now the reverse is also true. One of them has to be defined first unless we make the names static somehow.

**Review by GuillaumeRx — APPROVED** [review]

**Review by hmalik88 — APPROVED** [review]
LGTM

**Review by david0xd — APPROVED** [review]

**Comment by bowensanders** [comment]
QA: Manual and Automated tests pass for Chrome and Firefox [Flask].

**Pull request merged**